### DISCUSSION TITLE
Does Github Education still supply Unity Pro

### DISCUSSION THREAD

**Discussion by BigDreamerDev** [community/community#41847] [GitHub Education] [answered]
I got Github education but I still couldn't find the Unity benefit. Does it still exist?

**Comment by quzanh1130** [comment]
idk man. i cant help this one :)

**Comment by tigran-sargsyan-w** [comment, marked as answer]
Unfortunately this is no longer available. I also used this option before but now unity pro is only available to paid subscribers
